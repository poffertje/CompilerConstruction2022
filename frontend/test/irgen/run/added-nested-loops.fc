extern int printf(char[] format, ...);

int main() {
    for (int i = 0 to 10) {
        for (int j = 0 to 10) {
            if (i < 8) {
                break;
            }

            int k = i + j;
            while (k < 15) {
                if (k == i || k == j || k % 2 == 0){
                    k += 2;
                    continue;
                }
                printf("%d \n", k);
                k += 1;
            }
        }
    }
    return 0;
}