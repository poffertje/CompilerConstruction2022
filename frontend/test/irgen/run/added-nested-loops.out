9 
11 
13 
11 
11 
13 
